vendor/knative.dev/eventing-kafka/test/upgrade/continual.go

```diff
@@ -67,6 +67,22 @@ func ChannelContinualTest() pkgupgrade.BackgroundOperation {
 		"ChannelContinualTest", setup, verify)
 }
 
+// SourceContinualTest tests source operation in continual manner during the
+// whole upgrade and downgrade process asserting that all event are propagated
+// well.
+func SourceContinualTest() pkgupgrade.BackgroundOperation {
+	setup := func(c pkgupgrade.Context) {
+		// TODO: not yet implemented
```

Comment:
Now I think it would have been better to `Skip` the test because otherwise it looks like passing and people don't dive into "green" runs to check logs for warning. Most of the time. But now it's too late.

Reply:
Yup. We can fix that on upstream.

Reply:
If you're working on the test and will have it soonish we can wait for it.